Makefile

```diff
@@ -126,6 +126,7 @@ test-cmd: build
 # Example:
 #   make test-end-to-end
 test-end-to-end: build
+	hack/env hack/verify-generated-protobuf.sh # Test the protobuf serializations when we know Docker is available
```

Comment:
It does not check upstream protobuf, what do you mean by "additional
checks"?

On Fri, Jul 22, 2016 at 1:16 PM, David Eads <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/openshift/origin/pull/9814#discussion_r71913255>:
>
> > @@ -126,6 +126,7 @@ test-cmd: build
> >  # Example:
> >  #   make test-end-to-end
> >  test-end-to-end: build
> > +	hack/env hack/verify-generated-protobuf.sh # Test the protobuf serializations when we know Docker is available
>
> Is this wired to check the upstream ones too? Actually, given protobuf
> rules, we probably need additional checks to ensure that we're compatible,
> right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9814/files/f605a623322421bf84d1623a3037b41470c530d3..32e60e560f6d18abd70f9903411b694f01c2d539#r71913255>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p7bmzohK3-BKp6GOvCyUZtVCi2hRks5qYPsGgaJpZM4JLFc7>
> .
>